Assets/Python/ScreenInput.py

```
@@ -1,85 +1,93 @@
-	# Widget Option
-	def getOption (self):
-		return self.bOption
\ No newline at end of file
+class ScreenInput(object):
+    # __slots__ eliminates __dict__ overhead, saving ~100-200 bytes per instance
+    # Critical optimization for 32-bit memory constraints in games
+    __slots__ = (
+        'eNotifyCode', 'iData', 'uiFlags', 'iItemID', 'ePythonFileEnum',
+        'szFunctionName', 'bShift', 'bCtrl', 'bAlt', 'iMouseX', 'iMouseY',
+        'iButtonType', 'iData1', 'iData2', 'bOption'
+    )
```

Ruff (0.12.2)

`ScreenInput.__slots__` is not sorted

Apply a natural sort to `ScreenInput.__slots__`

(RUF023)